lib/hammer_cli_katello/content_view.rb

```diff
@@ -283,6 +283,10 @@ class AddContentViewVersionCommand < HammerCLIKatello::AddAssociatedCommand
       command_name 'add-version'
       desc _('Add a content view version to a composite view')
 
+      option "--content-view-id", "CONTENT_VIEW_ID",
+        _("Content view numeric identifier to search by"),
+        :attribute_name => :option_content_view_id
```

Comment:
I think might fix the test:

```suggestion
        :attribute_name => :option_content_view_version_content_view_id
```

Reply:
Ok, thanks for pointing that out!